src/StatusQ/Components/StatusListItemTag.qml

```diff
@@ -0,0 +1,77 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+Rectangle {
+    id: root
+    width: titleText.contentWidth + 60
+    height: 30
+    color: Theme.palette.primaryColor2
+    radius: 15
+
+    property string title: ""
+    signal click()
```

Comment:
would it make sense to rename this to "clicked"? just because when using the component, is more likely that someone types `onClicked`

Reply:
changed to clicked, habits with web but indeed qml use clicked